__tests__/main.test.ts

```diff
@@ -150,25 +151,25 @@ describe('Run Github Action', () => {
 
   describe('Handle configuration file', () => {
     test('Github Action fails if unable to parse config file', async () => {
-      const configPath = 'foobar'
+      const setFailedMock = jest.spyOn(core, 'setFailed').mockImplementation()
       process.env = {
         ...process.env,
-        INPUT_CONFIG_PATH: configPath,
+        INPUT_CONFIG_PATH: 'foo',
       }
-      await expect(run()).rejects.toThrowError('Config file not found')
-      expect(process.exitCode).toBe(1)
```

Comment:
We can't assert on this property, since changing it will make Jest return that exit code, and fail the CI.